LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright 2013-2014 Jack Naglieri <jacknagzdev@gmail.com>
```

Comment:
I think Dwolla has some interest in the copyright on the changes we're making, but I'm not sure how to express that here. Maybe this should be changed to 

```
Copyright 2013-2014 Jack Naglieri <jacknagzdev@gmail.com>, 2019 Dwolla
```

or something like that? Might need to get feedback from Legal.

Reply:
updated copyright on modifications made